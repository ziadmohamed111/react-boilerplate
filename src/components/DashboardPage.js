import React from 'react';

export const DashboardPage = () => {    
    return (
        (
            <div>
                Dashboard page content
            </div>
        )
    )
}

export default DashboardPage