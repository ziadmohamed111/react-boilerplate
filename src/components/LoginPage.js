import React from "react"
import { connect } from "react-redux"
import { startLogin } from "../actions/auth"

export const LoginPage = ( props ) => (
    <div className="box-layout">
        <div className="box-layout__login-box">
            <h1>Boiler plater</h1>
            <p>tag line for app</p>
            <button className="button-big" onClick={props.startLogin}>Login With Google</button>
        </div>
        
    </div>
)

const mapDispatchToProps = ( dispatch ) => ( {
    startLogin: () => dispatch( startLogin() )
} )

export default connect( undefined, mapDispatchToProps )( LoginPage ) 