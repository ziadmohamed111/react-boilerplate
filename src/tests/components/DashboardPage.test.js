import React from "react"
import { shallow } from "enzyme";
import { DashboardPage } from "../../components/DashboardPage"

test( "sholud render Dashboard Page correctly", () => {
    const wrapper = shallow( <DashboardPage /> )
    expect( wrapper ).toMatchSnapshot()
} )