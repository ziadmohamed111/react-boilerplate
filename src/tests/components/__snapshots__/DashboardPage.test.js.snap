// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sholud render Dashboard Page correctly 1`] = `
<div>
  Dashboard page content
</div>
`;
